src/test/java/edu/harvard/iq/dataverse/api/DatasetsIT.java

```diff
@@ -667,6 +667,60 @@ public void testCreatePublishDestroyDataset() {
         deleteDatasetResponse.prettyPrint();
         assertEquals(200, deleteDatasetResponse.getStatusCode());
 
+        // Start of deaccession test.
```

Comment:
Wow - the testCreatePublishDestroyDataset method is doing quite a bit more than testing destroy! +1 for splitting it up some day.